### PR TITLE
Allow Multiple Package Protocol Buffer Generation

### PR DESCRIPTION
**Pull request by bbengfort**
### Scope of changes

Modifies how we do protocol buffer generation to allow the go-ensign package to correctly generate protocol buffers. This modification is super annoying but I've worked on this for almost 3 hours now and I think this is the best we can do.

Fixes SC-14968

### Type of change

- [ ] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [x] technical debt
- [ ] other (describe)

### Acceptance criteria

Check to make sure this makes sense. Do we want a bash script like in the go-ensign repo? 

NOTE: this PR blocks https://github.com/rotationalio/go-ensign/pull/5

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [x]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: pkg/tenant/config/config.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ type AuthConfig struct {
 	KeysURL      string `split_words:"true" default:"https://auth.rotational.app/.well-known/jwks.json"`
 	Audience     string `default:"https://rotational.app"`
 	Issuer       string `default:"https://auth.rotational.app"`
-	CookieDomain string `split_words:"true" default:"https://rotational.app"`
+	CookieDomain string `split_words:"true" default:"rotational.app"`
```
Comment (rebeccabilbro) [inline pkg/tenant/config/config.go:50]: Intentional inclusion in this PR?

**Review by rebeccabilbro — APPROVED** [review]
I think this makes sense @bbengfort -- is the idea that we'll generate the protocol buffers here in `ensign` as well as in the SDK repos, and that in `ensign` we'll import the compiled pbs from `go-ensign` so we don't have a service definition conflict between the two versions of the compiled pbs?

File: docs/content/system/configuration.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ Tenant uses Quarterdeck to authenticate and authorize requests. The following co
 | TENANT_AUTH_KEYS_URL      | string | https://auth.rotational.app/.well-known/jwks.json | The path to the public keys used to verify JWT tokens.                              |
 | TENANT_AUTH_AUDIENCE      | string | https://rotational.app                            | The audience to verify inside of JWT tokens.                                        |
 | TENANT_AUTH_ISSUER        | string | https://auth.rotational.app                       | The issuer to verify inside of JWT tokens.                                          |
-| TENANT_AUTH_COOKIE_DOMAIN | string | https://rotational.app                            | Defines the domain that is set on cookies including CSRF cookies and token cookies. |
+| TENANT_AUTH_COOKIE_DOMAIN | string | rotational.app                                    | Defines the domain that is set on cookies including CSRF cookies and token cookies. |
```
[the remark this answers is not in the document]
Comment (bbengfort) [reply docs/content/system/configuration.md:110]: Intentional yes, just a minor update from the deployment stuff yesterday.

**Comment by bbengfort** [comment]
> I think this makes sense @bbengfort -- is the idea that we'll generate the protocol buffers here in `ensign` as well as in the SDK repos, and that in `go-ensign` we'll import the compiled pbs from `go-ensign` so we don't have a service definition conflict between the two versions of the compiled pbs?

Yes, we're trying to separate concerns so that `ensign` and `go-ensign` have independently generated protocol buffer code and that these scripts ensure they're kept up to date through the same generation process (trying to prevent recursive imports).

**Pull request merged**